source/excerptexport/templates/excerptexport/templates/extraction_order_status.html

```diff
@@ -0,0 +1,54 @@
+{% extends "excerptexport/layouts/base.html" %}
+
+{% block main %}
+    <nav>
+        <a href="{% url 'excerptexport:index' %}">&#8810; Home</a>
+    </nav>
+
+    <a href=""><button>&#8635; Reload</button></a>
+    <h2>Export status (excerpt &quot;{{ extraction_order.excerpt.name }}&quot;)</h2>
+    <table>
+        <body>
+            <tr>
+                <tr><th>ID</th><td>{{ extraction_order.id }}</td>
+                <tr><th>Excerpt</th><td>{{ extraction_order.excerpt.name }}</td>
+                <tr>
+                    <th>Status</th>
+                    <td>
+                        {% if extraction_order.get_state_display == 'UNDEFINED' %}
+                            <span style="color: maroon;">&quest;</span>
+                        {% elif extraction_order.get_state_display == 'INITIALIZED' %}
+                            <span style="color: cornflowerblue;">&bigstar;</span>
+                        {% elif extraction_order.get_state_display == 'WAITING' %}
+                            <span style="color: gold; display: inline-block; -ms-transform: rotate(90deg); -webkit-transform: rotate(90deg); transform: rotate(90deg);">&bowtie;</span>
+                        {% elif extraction_order.get_state_display == 'PROCESSING' %}
+                            <span style="color: olive;">&#8635;</span>
+                        {% elif extraction_order.get_state_display == 'FINISHED' %}
+                            <span style="color: seagreen;">&check;</span>
+                        {% elif extraction_order.get_state_display == 'CANCELED' %}
+                            <span style="color: maroon;">&Chi;</span>
+                        {% endif %}
+                        {{ extraction_order.get_state_display }}
+                    </td>
+                </tr>
+                {% if extraction_order.process_start_date %}
+                <tr><th>Extraction started at</th><td>{{ extraction_order.process_start_date|date:"r" }}</td>
+                {% endif %}
+                <tr><th>Issuer</th><td>{{ extraction_order.orderer.get_username }}</td></tr>
+        </body>
+    </table>
+
+    {% if extraction_order.get_state_display == 'FINISHED' %}
```

Comment:
Is it OK to have this level of business logic in django templates? (The knowledge that there are only downloads for finished orders isn't a pure view matter.) @njordan-hsr, how should one handle stuff like this?